drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/test/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/XLSBuilderAttributesTest.java

```diff
@@ -137,16 +137,16 @@ public void columnTitles() {
     public void content() {
 
         assertEquals("0", cell(9, 1).getStringCellValue());
-        assertEquals("09-Aug-2019", cell(9, 2).getStringCellValue());
-        assertEquals(true, cell(9, 3).getBooleanCellValue());
-        assertEquals(true, cell(9, 4).getBooleanCellValue());
-        assertEquals("09-Aug-2019", cell(9, 5).getStringCellValue());
+        assertEquals("\"09-Aug-2019\"", cell(9, 2).getStringCellValue());
+        assertEquals("true", cell(9, 3).getStringCellValue());
```

Comment:
So now will be the boolean attributes working?

Reply:
@jomarko yeah, now it uses text as the cell type so the text will read "true" in every case.